.github/ISSUE_TEMPLATE/update_truth.md

```diff
@@ -0,0 +1,60 @@
+---
+name: Update Truth
```

Comment:
Do we need to differentiate between updating the truth data for develop as opposed to updating the truth data for main_v5.1 for example? Or since the automated bot testing is only setup for develop, it that already obvious?

Reply:
The instructions to update the truth data do specify how to update for develop or a main branch. We don't have the mechanism to trigger main_vX.Y METplus runs from corresponding main_vX.Y branches in other METplus components implemented yet, but when we do then I think yes, we would need to differentiate the process a little more clearly. Right now it only applies to develop.